### PR TITLE
Add UdpClient Send for ReadOnlySpan

### PR DESCRIPTION
**Pull request by ngraziano**

### CODE REVIEW

File: Meziantou.Polyfill.Tests/UnitTest1.cs [hunk 1]
```diff
@@ -502,6 +504,38 @@ public void HttpContent_ReadAsStream()
         Assert.Equal([1, 2], streamContent.ToArray());
     }
 
+    [Fact]
+    public void UdpClient()
+    {
+        int port = 1000;
+
+        UdpClient GetPort()
```
Comment (meziantou) [inline Meziantou.Polyfill.Tests/UnitTest1.cs:512]:
```suggestion
        UdpClient CreateUdpClient()
```

File: Meziantou.Polyfill.Tests/UnitTest1.cs [hunk 2]
```diff
@@ -502,6 +504,38 @@ public void HttpContent_ReadAsStream()
...
+            while (true)
+            {
+                try
+                {
+                    return new UdpClient(port);
+                }
+                catch
+                {
+                    port++;
+                    if (port > 65000)
```
Comment (meziantou) [inline Meziantou.Polyfill.Tests/UnitTest1.cs:523]:
```suggestion
                    if (port >= ushort.MaxValue)
```

**Review by meziantou — COMMENTED** [review]
Thanks! Can you also improve the patch version in https://github.com/meziantou/Meziantou.Polyfill/blob/fd3da16053674c5832b3f985f9e6c4bd9d2e5e6d/Meziantou.Polyfill/Meziantou.Polyfill.csproj#L4 ?

**Comment by ngraziano** [comment]
I made the modification, I hope it's good.

**Comment by meziantou** [comment]
Thanks for your contribution!

**Pull request merged**